doc/conf.py

```diff
@@ -307,5 +307,7 @@
 autoclass_content = 'both'
 
 # Create Scenario-object to update the list of options in the docs
```

Comment:
That comment is no longer correct I assume.

Reply:
Removed the comment, code is self-explaining.